landing/views.py

```diff
@@ -1,8 +1,4 @@
-from django.shortcuts import render
+# from django.shortcuts import render
```

Comment:
Please remove this file

Reply:
@patrisiakaplun Is there a specific reason for not deleting?

Reply:
No reason, I've deleted them now